plugins/icon.rb

```diff
@@ -5,26 +5,28 @@ class Icon < Liquid::Tag
       def initialize(tag_name, args, tokens)
         super
         if args.strip =~ SYNTAX
-          @icon = Regexp.last_match(1).downcase
+          @icon = Regexp.last_match[:icon].downcase
+          @title = Regexp.last_match[:title]
```

Comment:
Should we generate a title from the icon automatically if it isn't set?

E.g., `mdi:bluetooth` resulting in `Bluetooth`?

Reply:
I like that

Reply:
works!